client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  Box,
+  InputWithLabelRow,
+  PositiveNumberInput,
+  TextWithLabelRow,
+  useTranslation,
+} from '@openmsupply-client/common';
+import {
+  LocationRowFragment,
+  Repack,
+  StockLineFragment,
+  useStock,
+} from '@openmsupply-client/system';
+import { LocationSearchInput } from 'packages/system/src/Location/Components/LocationSearchInput';
+import React, { FC, useState } from 'react';
+
+interface RepackEditFormProps {
+  invoiceId?: string;
+  stockLine: StockLineFragment | null;
+  onInsert: (repack: Repack) => void;
+}
+
+export const RepackEditForm: FC<RepackEditFormProps> = ({
+  invoiceId,
+  onInsert,
+  stockLine,
+}) => {
+  const t = useTranslation('inventory');
+  const { data } = useStock.repack.get(invoiceId ?? '');
+  const [location, setLocation] = useState<LocationRowFragment | null>(null);
+
+  return (
+    <Box display="flex" flexDirection="column" padding={2} gap={1}>
+      <TextWithLabelRow
+        label={t('label.pack-size')}
+        text={
+          invoiceId
+            ? String(data?.to.packSize ?? '')
```

Comment:
to since if there's invoiceId we want to show the repacked stock line? 🤔

Reply:
I thought it should show what was repacked from (at the top) and what it was repacked to (at the bottom). This would also be printable for warehouse staff to do that `operation`. I am not sure though what is meant to be shown in the changed UI since it's different to the original design:

![Screenshot 2023-06-01 at 10 56 03 PM](https://github.com/openmsupply/open-msupply/assets/26194949/5cc3ae7d-1699-49f6-9cd1-17c583c78a20)

vs

![Screenshot 2023-06-01 at 10 57 37 PM](https://github.com/openmsupply/open-msupply/assets/26194949/e6379543-d961-4441-a18d-8d493bb7a93f)

I got quite confused, `new number of packs` in the current implementation is actually number of packs to repack of the original pack, but mock design this was auto calculated and number of original packs to repack were shown at the top next to total number of packs at the time of repack (and that made sense because user is specifying number of original pack to repack into new pack size)

Based on the mock, when you open historic repack:

Pack Size = Out Invoice Line (from) -> pack size
Number Of Packs (left) = Out Invoice Line (from) -> number of packs
Number Of Packs (right) = nothing is shown, this would have come from stock line for new
Location = Out Invoice Line (from) -> Location

New Pack Size = In Invoice Line (to) -> pack size
New Location = In Inoivce Line (to) -> Location
New Number Of Packs = In Invoice Line (to) -> number of packs

Remainder is out of scope for now, since it would never be shown since we don't allow fractionals thus no remainders

Reply:
I'm very confused now... Also I thought remainder was the remainder from the original stock line...

Reply:
Will merge and move this to another issue (: